Source/Csla.Shared/Reflection/MethodCaller.cs

```diff
@@ -118,8 +118,10 @@ private static DynamicMethodHandle GetCachedMethod(object obj, string method, bo
 
     private static Dictionary<Type, DynamicCtorDelegate> _ctorCache = new Dictionary<Type, DynamicCtorDelegate>();
 
-    private static DynamicCtorDelegate GetCachedConstructor(Type objectType)
-    {
+    private static DynamicCtorDelegate GetCachedConstructor(Type objectType) {
```

Comment:
This might be picky, but putting the brace on the same line doesn't follow the conventions used in the CSLA codebase.

Reply:
I'll fix; following code style is important, I think R# just moved it back without me noticing.